tests/kafkatest/services/kafka/config_property.py

```diff
@@ -199,6 +200,9 @@
   val SSLEndpointIdentificationAlgorithmProp = SSLConfigs.SSL_ENDPOINT_IDENTIFICATION_ALGORITHM_CONFIG
   val SSLSecureRandomImplementationProp = SSLConfigs.SSL_SECURE_RANDOM_IMPLEMENTATION_CONFIG
   val SSLClientAuthProp = SSLConfigs.SSL_CLIENT_AUTH_CONFIG
+
+  /** New group coordinator configs */
+  val NewGroupCoordinatorEnableProp = "group.coordinator.new.enable"
```

Comment:
nit: Could we reuse NEW_GROUP_COORDINATOR_ENABLE here?

Reply:
This javadoc is just to mention what is present in KafkaConfig.scala, this is the string used there so I copied the same name here. They did the same thing for the rest of the properties too. I don't think the name has to match but the string does.